Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -672,9 +692,10 @@ extension NIOHTTP2Handler {
             self.failDroppedPromises(droppedPromises, streamID: streamClosedData.streamID, errorCode: streamClosedData.reason ?? .cancel)
         case .streamCreated(let streamCreatedData):
             self.outboundBuffer.streamCreated(streamCreatedData.streamID, initialWindowSize: streamCreatedData.localStreamWindowSize.map(UInt32.init) ?? 0)
-            self.inboundEventBuffer.pendingUserEvent(NIOHTTP2StreamCreatedEvent(streamID: streamCreatedData.streamID,
-                                                                                localInitialWindowSize: streamCreatedData.localStreamWindowSize.map(UInt32.init),
-                                                                                remoteInitialWindowSize: streamCreatedData.remoteStreamWindowSize.map(UInt32.init)))
+            self.inboundEventBuffer.pendingUserEvent(
+                NIOHTTP2StreamCreatedEvent(streamID: streamCreatedData.streamID,
+                                                          localInitialWindowSize: streamCreatedData.localStreamWindowSize.map(UInt32.init),
+                                                          remoteInitialWindowSize: streamCreatedData.remoteStreamWindowSize.map(UInt32.init)))
```

Comment:
Can you clean up the indentation here?

Reply:
I _think_ I have done this now. I have certainly changed it.